packages/sit-onyx/.storybook/managed.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  ArgTypesEnhancer,
+  InputType,
+  SBType,
+  StrictInputType,
+  walkTree,
+} from "@sit-onyx/storybook-utils";
+
+const SB_TYPE_CONTROL_MAP = new Map<SBType["name"], InputType["control"]>([
```

Comment:
Why is this a map instead of a regular object?

Reply:
No particular reason, replaced it with an object literal.